packages/synapse-react-client/src/lib/containers/ExperimentalMode.tsx

```diff
@@ -60,9 +56,21 @@ const ExperimentalMode: React.FC = () => {
       >
         Experimental mode is {isExperimentalModeOn ? 'on' : 'off'}
       </Button>
-      <OverlayTrigger trigger="click" placement="top" overlay={popover}>
-        <InfoOutlined style={{ verticalAlign: 'middle' }} />
-      </OverlayTrigger>
+      <LightTooltip
```

Comment:
Used a light tooltip that appears on clicking the info icon to replicate the current design. We could consider using a dark tooltip that appears on hover to match the tooltip on Version in the Portals footer (i.e. where this component is used)

Reply:
Design has typically used the dark tooltip for 1-2 sentences, where you'd typically hover over a button to see it. For longer content, they would typically use a light popover (they call it a "super tooltip") opened by clicking an info button. The "super tooltip" may also contain buttons where we may link out to documentation.

See the `MarkdownPopover` component for the "super tooltip" case.

I like the idea of just using the regular (dark) tooltip here. The content isn't super long or interactive, so it would probably be appropriate.

You could consider keeping your LightTooltip to see if it's a better foundation for `MarkdownPopover`

Reply:
Thanks Nick! Good to know about the "super tooltip" -- I think the LightTooltip will be helpful since MUI Popvers don't come with [an arrow](https://github.com/mui/material-ui/issues/18928) by default. Happy to switch to the dark tooltip here!